css/style.css

```diff
@@ -0,0 +1,651 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+  min-height: 100%;
+}
+
+body {
+  min-width: 375px;
+  background-color: #fff;
+  font-family: 'Lato', sans-serif;
+  color: #fff;
+}
+
+/* ------------------------------- Header Section ----------------------------- */
+
+header {
+  margin: 0;
+  top: 0;
+}
+
+#homepage {
+  background-image: url(/images/hero-image-mobile.png);
+  background-repeat: no-repeat;
+  height: 90vh;
+  background-position: center;
+  background-size: cover;
+}
+
+.top-nav-bar {
+  display: none;
+}
+
+.nav-bar-desktop {
+  display: none;
+}
+
+.nav-bar-mobile {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 7% 10%;
+}
+
+.mobile-nav {
+  display: flex;
+  justify-content: left;
+  width: 100%;
+}
+
+.mobile-menu {
+  display: block;
+  padding: 10%;
+  position: fixed;
+  left: 0;
+  top: 0;
+  background-color: #272a31;
+  height: 100vh;
+  width: 100vw;
+  z-index: 9;
+  opacity: 0.8;
+  transition: opacity 2s, visibility 2s;
+}
+
+.inactive {
+  display: none;
+}
+
+.active {
+  display: block;
+}
+
+.mobile-menu ul {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.mobile-menu li {
+  margin: 4% 20%;
+}
+
+.close-btn {
+  display: flex;
+  justify-content: flex-end;
+  margin: 5% 30%;
+  padding: 0 0;
+}
+
+.mobile-menu a {
+  color: #fff;
+  text-decoration: none;
+  font-size: 30px;
+}
+
+.menu-icon {
+  font-size: 30px;
+  color: #fff;
+}
+
+.hero-content {
+  padding: 5%;
+}
+
+.hero-content h1 {
+  font-family: cocogoose, sans-serif;
+  color: #ec5242;
+  font-weight: 700;
+}
```

Comment:
- [ ] If you observe very closely, you can notice that the heading has an inside image. Kindly ensure that your heading matches the original design provided to have a professional application with the best UI ☺️.
- [ ] In addition, kindly increase the text size so that the application matches the original design

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/232789345-70d9e242-3445-4ca2-a190-bfe0b6286820.png)

Original design 😎 
![image](https://user-images.githubusercontent.com/99394374/232789457-b0df6392-f36b-4ddd-b715-989163b0cfdc.png)